cmd/destroy.go

```diff
@@ -0,0 +1,64 @@
+// Package cmd for handling command line arguments
+package cmd
+
+import (
+	"os"
+	"os/exec"
+
+	"github.com/rawdaGastan/gridify/internal/config"
+	"github.com/rawdaGastan/gridify/internal/deployer"
+	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
+	"github.com/spf13/cobra"
+)
+
+// destroyCmd represents the destroy command
+var destroyCmd = &cobra.Command{
+	Use:   "destroy",
+	Short: "Destroy the project in the current directory from threefold grid",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		config, err := config.LoadConfigData()
```

Comment:
same as the rest, please make a separate function in e.g internal/cmd/destroy thath as the logic and only invoke it here with the required params